force-app/main/default/lwc/propertyFilter/__tests__/propertyFilter.test.js

```diff
@@ -0,0 +1,238 @@
+        );
+
+        // Run all fake timers.
+        jest.runAllTimers();
+
+        const SEARCH_CRITERIA = {
+            searchKey: '',
+            maxPrice: 60000,
+            minBedrooms: 0,
+            minBathrooms: 0
+        };
+        return Promise.resolve().then(() => {
+            // Was publish called and was it called with the correct params?
+            expect(publish).toHaveBeenCalledWith(
+                undefined,
+                FILTERSCHANGEMC,
+                SEARCH_CRITERIA
+            );
+        });
+    });
+
+    it('fires the change event on Bedrooms slider input', () => {
+        // Create initial element
+        const element = createElement('c-property-filter', {
+            is: PropertyFilter
+        });
+        document.body.appendChild(element);
+
+        // Query lightning-input element
+        const lightningSliderEl = element.shadowRoot.querySelectorAll(
+            'lightning-slider'
+        )[1];
+        lightningSliderEl.value = 2;
+        lightningSliderEl.dispatchEvent(
+            new CustomEvent('change', {
+                detail: {
+                    value: 2
+                }
+            })
+        );
+
+        // Run all fake timers.
+        jest.runAllTimers();
+
+        const SEARCH_CRITERIA = {
+            searchKey: '',
+            maxPrice: MAX_PRICE,
+            minBedrooms: 2,
+            minBathrooms: 0
+        };
+        return Promise.resolve().then(() => {
+            // Was publish called and was it called with the correct params?
+            expect(publish).toHaveBeenCalledWith(
+                undefined,
+                FILTERSCHANGEMC,
+                SEARCH_CRITERIA
+            );
+        });
+    });
+
+    it('fires the change event on Bathrooms slider input', () => {
+        // Create initial element
+        const element = createElement('c-property-filter', {
+            is: PropertyFilter
+        });
+        document.body.appendChild(element);
+
+        // Query lightning-input element
+        const lightningSliderEl = element.shadowRoot.querySelectorAll(
+            'lightning-slider'
+        )[2];
+        lightningSliderEl.value = 2;
+        lightningSliderEl.dispatchEvent(
+            new CustomEvent('change', {
+                detail: {
+                    value: 2
+                }
+            })
+        );
+
+        // Run all fake timers.
+        jest.runAllTimers();
+
+        const SEARCH_CRITERIA = {
+            searchKey: '',
+            maxPrice: MAX_PRICE,
+            minBedrooms: 0,
+            minBathrooms: 2
+        };
+        return Promise.resolve().then(() => {
+            // Was publish called and was it called with the correct params?
+            expect(publish).toHaveBeenCalledWith(
+                undefined,
+                FILTERSCHANGEMC,
+                SEARCH_CRITERIA
+            );
+        });
+    });
+
+    it('resets to default values when reset button is clicked', () => {
```

Comment:
This test isn't doing what it says: you're not checking those default values, you're just checking for the publish event.

Checking for the event is good so keep that and rename this test to "fires change event when reset button is clicked".

Add also an extra test that checks that `handleReset()` resets the input and slider values. This one will indeed be named "resets to default values when reset button is clicked"